lib/features/about/author.info.table.widget.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:guess_the_text/features/about/contributor.model.dart';
+
+import '../../theme/widgets/text.link.widget.dart';
+
+class AuthorInfoTable extends StatefulWidget {
+  const AuthorInfoTable({Key? key}) : super(key: key);
+
+  @override
+  State<AuthorInfoTable> createState() => _AuthorInfoTableState();
+}
+
+class _AuthorInfoTableState extends State<AuthorInfoTable> {
+  final List<Contributor> _contributors = <Contributor>[
```

Comment:
Suggestion:
- since this is not really a state than can change, you can convert the widget to be a [StatelessWidget](https://api.flutter.dev/flutter/widgets/StatelessWidget-class.html) which will be optimized